.github/workflows/operatorBuildAndDeploy.yml

```diff
@@ -40,35 +40,143 @@ jobs:
         run: |
           export PATH=$GOROOT/bin:$GOPATH/bin:$PATH
           mage operator:testGenerateClient
-      - name: Build docker - standard and ubi images
+      - name: Unit Tests
+        run: |
+          export PATH=$GOROOT/bin:$GOPATH/bin:$PATH
+          mage operator:testGo
+      - name: Login to GitHub Package Registry
+        run: echo "${{ secrets.GITHUB_TOKEN }}" | docker login docker.pkg.github.com -u "${{ github.actor }}" --password-stdin
+      - name: Login to ECR
         env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.ECR_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.ECR_SECRET }}
+        run: $(aws ecr get-login --no-include-email --region us-east-1)
+      - name: Setup Buildx
+        id: buildx
+        uses: crazy-max/ghaction-docker-buildx@v3
+      - name: Cache Docker layers
+        uses: actions/cache@v2
+        id: cache
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildx-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-
+      - name: Build and Push Docker
+        if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name == 'datastax/cass-operator'
+        env:
+          ECR_REPO: ${{ secrets.ECR_REPO }}
```

Comment:
looks like this secret is named wrong - ECR_REGISTRY would be better

Reply:
Technically, it should be registry I think. I cannot change the secret names though or add new ones on this repo or the config builder repo.

Reply:
yeah, better to not fuss with it, maybe I'll leave a comment on it in another PR